src/main/java/com/cloudbees/jenkins/plugins/sshagent/mina/MinaRemoteAgentFactory.java

```diff
@@ -66,7 +68,10 @@ public RemoteAgent start(Launcher launcher, final TaskListener listener) throws
         return launcher.getChannel().call(new MinaRemoteAgentStarter(listener));
     }
 
-    private static class TomcatNativeInstalled implements Callable<Boolean, Throwable> {
+    private static class TomcatNativeInstalled extends MasterToSlaveCallable<Boolean, Throwable> {
+
+        private static final long serialVersionUID = 3234893369850673438L;
```

Comment:
:ant: normally I would include a comment to explain that this was generated from the 1.7 version of the class and also include a `@since 1.8` javadoc comment